core/domain/wipeout_service.py

```diff
@@ -243,6 +286,50 @@ def pre_delete_user(user_id):
     save_pending_deletion_requests(pending_deletion_requests)
 
 
+def run_user_deletion(pending_deletion_request):
+    """Run the user deletion.
+
+    Args:
+        pending_deletion_request: PendingDeletionRequest. The domain object of
+            the user being deleted.
+
+    Returns:
+        str. The outcome of the deletion.
+    """
+
+    if pending_deletion_request.deletion_complete:
+        return wipeout_domain.USER_DELETION_ALREADY_DONE
+    else:
+        delete_user(pending_deletion_request)
+        pending_deletion_request.deletion_complete = True
+        save_pending_deletion_requests([pending_deletion_request])
+        return wipeout_domain.USER_DELETION_SUCCESS
+
+
+def run_user_verification(pending_deletion_request):
+    """Run the user deletion verification.
+
+    Args:
+        pending_deletion_request: PendingDeletionRequest. The domain object of
+            the user being verified.
+
+    Returns:
+        str. The outcome of the verification.
+    """
+
+    if not pending_deletion_request.deletion_complete:
+        return wipeout_domain.USER_VERIFICATION_NOT_DELETED
+    elif verify_user_deleted(pending_deletion_request):
+        delete_pending_deletion_request(pending_deletion_request.user_id)
+        email_manager.send_account_deleted_email(
```

Comment:
Verifying a deletion seems like it should not be a state-changing action -- in particular, I wouldn't have expected it to send an email. So the name/framing might need to be changed.

In addition, why can't the verification be run and the email sent automatically (why must it be triggered through the admin tab)?

Reply:
Any idea for the naming? Maybe something like `CompleteDeletion`?

It will either be run by the cron or can be run manually from the admin page (which will be mostly used in the e2e tests).

Reply:
Done.